src/web-ui/src/public/DemoGuideServiceDetails.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class="demo-guide col-sm-12 col-md-9 col-lg-9 mb-4 about-service" style="padding:0">
+        <div v-for="subsection in subsections" v-bind:key=subsection.name style="padding:1em;" :style="{display: displaysection === subsection.name ? 'block' : 'none' }">
```

Comment:
Any specifc reason to use styles inline here?
Can we move all the styling to the bottom of the file under <style scoped>?

Reply:
It would actually create a lot more classes/divisions since many sections require a specific style - hence preferred putting it inline instead of creating that many classes.

Reply:
How many are we talking about? I see many cases where inline styles can be replaced with Bootstrap classes. Also the inconsistency of using scoped template CSS with inline styling defeats the purpose of CSS and makes the templates harder to read and maintain.

Reply:
Makes sense. I will try to reduce as many as possible.